operator/images/access-setup/content/bin/setup_work_dir.sh

```diff
@@ -172,6 +172,14 @@ tekton_chains_manifest(){
 }
 
 tekton_results_manifest(){
+  miniosecret="$(kubectl get secrets -n tekton-results | grep "minio-storage-configuration" || true )"
+  if [ -z "$miniosecret" ]; then
```

Comment:
Why not `if ! $(kubectl get secrets -n tekton-results minio-storage-configuration >/dev/null); then`

Reply:
that does not work for me when I try it locally

simplifying to `kubectl get secrets minio-storage-configuration -o name -n tekton-results  --ignore-not-found` and setting that to `miniosecret` works for me

you good with that ?

Reply:
I've pushed ^^ up in the interim in a separate commit

I'll squash once we are sorted

Reply:
worked in the test
```
      - tekton-results manifest:
        ++ kubectl get secrets minio-storage-configuration -o name -n tekton-results --ignore-not-found
        + miniosecret=secret/minio-storage-configuration
        + '[' -z secret/minio-storage-configuration ']'
        + printf 'Tekton results secrets already in place, returning from tekton_results_manifest \n'
        Tekton results secrets already in place, returning from tekton_results_manifest 
```

Reply:
I'm good with it.

Reply:
cool / thanks